ethereum/statetransition/src/test/java/tech/pegasys/teku/statetransition/validation/AggregateAttestationValidatorTest.java

```diff
@@ -219,31 +198,12 @@ public void shouldSaveForFutureWhenStateIsNotAvailable() {
         ValidateableAttestation.aggregateFromValidator(spec, aggregate);
     when(attestationValidator.singleOrAggregateAttestationChecks(
             any(), eq(attestation), eq(OptionalInt.empty())))
-        .thenReturn(SafeFuture.completedFuture(InternalValidationResult.SAVE_FOR_FUTURE));
+        .thenReturn(SafeFuture.completedFuture(InternalValidationResultWithState.saveForFuture()));
 
     assertThat(validator.validate(attestation))
         .isCompletedWithValue(InternalValidationResult.SAVE_FOR_FUTURE);
   }
 
-  @Test
-  public void shouldRejectWhenAttestationValidatorSavesForFutureAndAggregateChecksFail() {
```

Comment:
Removed tests as it doesn't make sense to load a state to process the aggregate when it can't be accepted now. We'll wind up doing the full validation again when we process in the future so just leave it to then.